openhcl/openhcl_boot/src/main.rs

```diff
@@ -123,8 +123,12 @@ fn build_kernel_command_line(
         "panic_print=0",
         // RELIABILITY: Reboot immediately on panic, no timeout.
         "panic=-1",
-        // RELIABILITY: Print processor context information on a fatal signal.
-        "print_fatal_signals=1",
+        // RELIABILITY: Don't print processor context information on a fatal
+        // signal. Our crash dump collection infrastructure seems reliable, and
+        // this information doesn't seem useful without a dump anyways.
+        // Additionally it may push important logs off the end of the kmsg
+        // page logged by the host.
+        "print_fatal_signals=0",
```

Comment:
This is the default, so this is just noise. I'd leave it out.

Reply:
(If you want to leave it commented out so that we have the context, that's OK. Or just remove the whole thing and maybe send Roman a heads up.)